modules/@apostrophecms/module/index.js

```diff
@@ -74,6 +75,11 @@ module.exports = {
     // Routes in their final ready-to-add-to-Express form
     self._routes = [];
 
+    // Add structured logging if we passed the util module.
```

Comment:
I think you mean: "Enable structured logging if this is not the util module itself"

Reply:
Or `log` or `util` :)